src/library/scala/reflect/Manifest.scala

```diff
@@ -80,14 +80,11 @@ abstract class AnyValManifest[T <: AnyVal](override val toString: String) extend
   override val hashCode = System.identityHashCode(this)
 }
 
-/** `ManifestFactory` defines factory methods for manifests.
- *  It is intended for use by the compiler and should not be used in client code.
- *
- *  Unlike `Manifest`, this factory isn't annotated with a deprecation warning.
- *  This is done to prevent avalanches of deprecation warnings in the code that calls methods with manifests.
- *  Why so complicated? Read up the comments for `ClassManifestFactory`.
+/** Defines factory methods for manifests.
  */
-object ManifestFactory {
+object Manifest extends ManifestFactoryImpl
+object ManifestFactory extends ManifestFactoryImpl  // TODO: remove after next re-STARR
```

Comment:
This is very dangerous. That means there will be two identities for the classtag of `Byte` (`Manifest.Byte` and `ManifestFactory.Byte`) and similarly for all the other special classtags. This is *bound* to break some code.

Reply:
but only until next re-STARR! after re-STARRing, we could get rid of `ManifestFactoryImpl`  and land on just having `object Manifest`

Reply:
Hum, I see. Yes, that could work, assuming not too many tests break right now.

Reply:
you think it's dangerous to have even briefly? I guess CI will tell us